src/Console/PushCommand.php

```diff
@@ -74,13 +78,13 @@ protected function pushSynonyms($indexName)
         $task = $index->clearSynonyms(true);
         $index->waitTask($task['taskID']);
 
-        $synonyms = Json::decode(File::get($this->path.$indexName.'-synonyms.json'), true);
+        $synonyms = Json::decode(File::get($this->indexRepository->getFilePath($indexName, 'synonyms')), true);
```

Comment:
Here for instance, it would be great if we could have `$this->indexRepository->getSynonyms` instead of having to read the file in the command.